internal/venice-common/src/main/java/com/linkedin/venice/utils/BlobTransferCommonUtils.java

```diff
@@ -0,0 +1,11 @@
+package com.linkedin.venice.utils;
+
+public class BlobTransferCommonUtils {
+  /**
+   * Enum representing the configuration type for blob transfer in the server.
+   * Put this in the common utils package so that it can be used in both router, server and controller.
+   */
+  public enum BlobTransferInServerConfigType {
+    NOT_SPECIFIED, ENABLED, DISABLED
```

Comment:
Yeah again referring to my earlier comment.  A lot of configs in store config are boolean based.  I think if at all possible we should stick to that pattern.